### PR TITLE
feat: implement save zklend event to db

### LINKED ISSUE #258: Save Zklend event to db
1.  Read our [Contributor Guidelines](https://github.com/CarmineOptions/derisk-research/blob/master/CONTRIBUTING.md) and [README](https://github.com/CarmineOptions/derisk-research/blob/master/README.md).
2. Add new file `transform_events.py` [here](https://github.com/CarmineOptions/derisk-research/tree/master/apps/data_handler/handlers/events/zklend)
3. Create new class `ZkLendTransformer`
4. Parse events using [DeriskAPIConnector](https://github.com/CarmineOptions/derisk-research/blob/master/apps/data_handler/handler_tools/api_connector.py#L9)
5. To get credentials, text me in DM @djeck_vorobey1
6. After fecthing event, create mapping: `event_name`:(`serializer_class`, `model_class`)
Example: `{"AccumulatorsSync":(AccumulatorsSyncEventData, AccumulatorsSyncEventData)}`  
7. Models you can find [here](https://github.com/CarmineOptions/derisk-research/blob/master/apps/data_handler/db/models/zklend_events.py)
8. Serializers you can find [here](https://github.com/CarmineOptions/derisk-research/blob/master/apps/data_handler/handler_tools/data_parser/serializers.py#L7)
9. Parse appropriate events in `ZkLendTransformer` and save it to db

### PR DESCRIPTION
**Pull request by mayasimi**
Fixes #258

### CODE REVIEW

**Comment by mayasimi** [comment]
@djeck1432 ready for review

**Comment by mayasimi** [comment]
@djeck1432 kindly review

File: apps/data_handler/handlers/events/zklend/transform_events.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+class ZklendTransformer:
+    """
+    A class that is used to transform Zklend events into database models.
+    """
+
+    EVENT_MAPPING: Dict[str, Tuple[Callable, Type[BaseModel], Type[Base]]] = EVENT_MAPPING
+    
+    def __init__(self):
```
Comment (djeck1432) [inline apps/data_handler/handlers/events/zklend/transform_events.py:41]: add docstring please

File: apps/data_handler/handlers/events/zklend/transform_events.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    def save_liquidation_event(self, event: Dict[str, Any]) -> None:
+        """
+        Save a liquidation event to the database.
+        """
+        parser_func, serializer_class, model_class = EVENT_MAPPING["Liquidation"]
+        parsed_data = parser_func(event)
+        db_model = model_class(**parsed_data.model_dump())
+        
+        self.db_connector.create_liquidation_event(db_model)
```
Comment (djeck1432) [inline apps/data_handler/handlers/events/zklend/transform_events.py:62]: Please ,add `if __name__ == "__main__"` to test it

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, add also logic to fetch `ZkLend` address and block_number and send it to DersikApiConnector to finalize this task

**Comment by djeck1432** [comment]
@mayasimi Please, add also logic to fetch ZkLend address and block_number and send it to DersikApiConnector to finalize this task

**Comment by mayasimi** [comment]
> Please, add also logic to fetch `ZkLend` address and block_number and send it to DersikApiConnector to finalize this task

@djeck1432 where can i fetch the addresses and block numbers from

File: apps/data_handler/handlers/events/zklend/transform_events.py (hunk 1 above)
Comment (djeck1432) [reply apps/data_handler/handlers/events/zklend/transform_events.py:62]: Add docstring please here.

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**